vkcs/internal/acctest/acctest.go

```diff
@@ -195,3 +197,14 @@ func ImportStep(resourceName string) resource.TestStep {
 		ImportStateVerify: true,
 	}
 }
+
+func GenerateTags(testName string) map[string]string {
+	return map[string]string{
+		"TestName":    testName,
+		"CurrentTime": fmt.Sprint(time.Now().Unix()),
+	}
+}
+
+func GenerateClusterNameSuffix() string {
```

Comment:
It's just a random suffix that can be used anywhere, so, maybe, it should be named less specific?

Reply:
change to GenerateNameSuffix